slices/perl-base.yaml

```diff
@@ -0,0 +1,32 @@
+package: perl-base
+
+essential:
+  - perl-base_copyright
+
+slices:
+  modules:
```

Comment:
It's possible, but I doubt it would be too fragmented. The functional modules there include `autoloader`, `carp`, `config`, `cwd`, `dynaloader`, `errorno`, `exporter`, `fcntl`, `file`, `geopt`, `hash`, `io`, `ipc`, `list`, `posix`, `scalar`, `selectsaver`, `socket`, `symbol`, `text`, `tie`, `xsloader`, `attributes`, `buildin`, `bytes`, `constant`, `feature`, `fields`, `integer`, `lib`, `locale`, `overload`, `parent`, `re`, `strict`, `auto` and `unicore`. Among these modules, `auto` is the module libraries automatically imported by perl if running any of the other functional modules that require their corresponding `.so` library. The `unicore` contains the core unicode support for perl. All these functional modules seem to be the very basic and fundamental modules of perl, and therefore we should not split them further.

Reply:
So, if I got it right:
 - `auto` (and maybe `unicore`) should be a dependency of every other module. 

But then, the other modules seem able to live independently from each other, don't they? Why not:

```yaml
error-handling:
  - carp
  - errorno
  - warnings

file-handling:
  - cwd
  - file
  - io
  - fcntl

networking:
  - socket
  - ipc

(etc)
```

And you still keep the `modules`, but it is just a list of dependencies